src/SFA.DAS.AssessorService.Web/Validators/Standard/OptInStandardVersionViewModelValidator.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Linq;
+using FluentValidation;
+using Microsoft.AspNetCore.Http;
+using SFA.DAS.AssessorService.Application.Api.Client.Clients;
+using SFA.DAS.AssessorService.Web.Helpers;
+using SFA.DAS.AssessorService.Web.ViewModels.Standard;
+
+namespace SFA.DAS.AssessorService.Web.Validators.Standard
+{
+    public class OptInStandardVersionViewModelValidator : AbstractValidator<OptInStandardVersionViewModel>
+    {
+        public OptInStandardVersionViewModelValidator(IHttpContextAccessor httpContextAccessor, IStandardVersionClient standardVersionApiClient)
+        {
+            var epaoOrgId = EpaOrgIdFinder.GetFromClaim(httpContextAccessor);
+
+            RuleFor(vm => vm.StandardReference)
+                .NotEmpty()
```

Comment:
The check for empty should not be done should rely on the exceptions for that in the action method

Reply:
Removed